sktime/clustering/partitioning/_lloyds.py

```diff
@@ -240,6 +240,11 @@ def __init__(
 
         self._random_state = None
         self._init_algorithm = None
+        self._initial_cluster_centers = None
+
+        if isinstance(self.init_algorithm, np.ndarray):
+            self._initial_cluster_centers = self.init_algorithm
+            self.init_algorithm = lambda *args, **kwargs: self._initial_cluster_centers
```

Comment:
I think this should be `self._init_algorithm` (with a leading underscore).

Also, to avoid spreading this logic out, either move this into `_check_params`, or move the `_init_algorithm` logic from `_check_params` here.

Reply:
Moved the logic to `_check_params` in commit 109bbc3.